exonum/src/storage/list_index.rs

```diff
@@ -581,6 +639,23 @@ mod tests {
             let mut list_index = ListIndex::new_in_family(IDX_NAME, &vec![01], &mut fork);
             super::list_index_iter(&mut list_index);
         }
+
+        #[test]
+        fn list_index_clear_in_family() {
+            const PARAMS: &[(u32, u32, bool)] = &[
+                (0_u32, 5_u32, false),
+                (5_u32, 0_u32, false),
+                (1_u32, 7_u32, true),
+                (7_u32, 1_u32, true),
+            ] as &[(u32, u32, bool)];
```

Comment:
3 different ways of type indication is used. Is it really needed?

Reply:
Agree with @vitvakatu. Parameters types in the tuple is enough I suppose.

Reply:
IIRC, `const` in Rust requires explicit type specification so we can leave only that:

```rust
const PARAMS: &[(u32, u32, bool)] = &[
    (0, 5, false),
    // ...
];
```

Reply:
I *think* both explicit `_u32` suffixes and explicit cast to slice were required for the code to compile in Rust 1.27 (I used it to compile the code initially before seeing that we require 1.30 now), but apparently, 1.30 doesn't require them. I'm not going to install 1.27 back to check tho, so maybe it's just a brainfart on my side 🙃

Reply:
Travis build has a job to test 1.27 compatibility.

I have 1.27.2 installed and this compiles fine with it:

```rust
fn main() {
    const THINGS: &[(u32, u32, bool)] = &[
        (0, 0, false),
        (9, 20, true),
    ];

    for stuff in THINGS {
        println!("{:?}", stuff);
    }
}
```